src/main/java/GUI/FootballMain.java

```diff
@@ -272,12 +279,15 @@ public static void showDialog(UI lastUI, String msg, String receiveType,StringBu
             newWindow.setCloseOnOutsideClick(false);
             newWindow.setCloseOnEsc(false);
             newWindow.setVisible(true);
+            newWindow.setWidth("500px");
```

Comment:
Why did you set this to specified size? 
you can just add a label and the label will get the max width it needs

Reply:
It's not working without it